src/arviz_plots/plot_collection.py

```diff
@@ -1019,8 +1019,11 @@ def map(
 
         for var_name, sel, isel in plotters:
             da = data[var_name].sel(sel)
-            if np.all(np.isnan(da)):
-                continue
+            try:
+                if np.all(np.isnan(da)):
+                    continue
+            except TypeError:
+                pass
```

Comment:
with none backend the elements in `da` can be lists or dicts, so isnan errors out. This allows the new `render` function to work through `.map` and I think the behaviour is still exactly what we want: if everything is exactly nan do nothing, otherwise go ahead and call the function which should work with whatever it gets